requirements.txt

```diff
@@ -1,6 +1,7 @@
 aspy.yaml==1.3.0
 beautifulsoup4==4.8.1
 bs4==0.0.1
```

Comment:
Unrelated, but we should be using `beautifulsoup4` instead

![image](https://user-images.githubusercontent.com/8889311/73772735-bbae3980-474e-11ea-90f9-55586eef9004.png)